tests/platform_tests/bids-apps_BIDS_multi_session_session_ses-Session2.py

```diff
@@ -0,0 +1,86 @@
+#! /usr/bin/env python3
+"""Run bids-freesurfer on session "ses-Session2"
+
+    This script was created to run Job ID 61672224ed5fcc2ee72d8458
+    In project "bids-apps/BIDS_multi_session"
+    On Flywheel Instance https://ga.ce.flywheel.io/api
+"""
+
+import os
+import argparse
+from datetime import datetime
+
+
+import flywheel
+
+
+input_files = {}
+
+
+def main(fw):
+
+    gear = fw.lookup("gears/bids-freesurfer")
+    print("gear.gear.version for job was = 1.0.4_6.0.1-5")
```

Comment:
"gear.gear.version for originally generated test job was" ?

Reply:
Good idea!